shopinvader/services/customer.py

```diff
@@ -114,3 +159,52 @@ def _prepare_create_response(self, binding):
         response = self._assign_cart_and_get_store_cache()
         response["data"] = {"id": self.partner.id, "name": self.partner.name}
         return response
+
+    def _validator_return_create(self):
+        schema = self._schema_for_assign_cart_and_get_store_cache()
+        schema.update(
+            {
+                "data": {
```

Comment:
question, cannot find documentation for this, why it is wrapped to "data" key?  for schema we usually return only dict with keys